feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -176,6 +192,9 @@ export class CustomerTicketingService implements CustomerTicketingFacade {
     return this.getTicketState().pipe(map((state) => state.data));
   }
 
+  createTicket(ticket: TicketStarter): Observable<TicketStarter | unknown> {
+    return this.createTicketCommand.execute(ticket);
```

Comment:
On the one hand, when we create a Ticket, the POST endpoint returns a TicketDetails entity, not a Ticket Starter.
You might not use the Ticket since you might trigger a server call upon creation, but just for correctness.
On the other hand, can we have ticketStarted instead of ticket to avoid confusions